telemetry.go

```diff
@@ -0,0 +1,31 @@
+package gocron
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	jobLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "gocron_job_latency_seconds",
+		Help: "The latency of jobs in seconds.",
+	}, []string{"job_name"})
```

Comment:
might want to optionally include a job_name_prefix or scheduler_name_prefix option that could be set when enabling metrics on the scheduler to prefix all metric labels on the scheduler with something.